kythe/go/extractors/config/extractor.go

```diff
@@ -40,7 +43,9 @@ type Repo struct {
 	OutputPath string
 	// An optional path to a file containing a
 	// kythe.proto.ExtractionConfiguration encoded as JSON that details how
-	// to perform extraction.
+	// to perform extraction. If this is unset, the extractor will first try
+	// to find a config defined in the repo, or finally use a hard coded
```

Comment:
I wonder: If we're going to use a hardcoded default config anyway, could we maybe just bake it in to the binary directly? E.g., generate it and include it in the package as static data?

Reply:
discussed offline:
- my initial code here was baking it in as native go (build up the pb from scratch), but that was clunky as hell
- settling on middle ground here to specify the string json text proto in a .go file, and have that .go file expose a strings.Reader. 

This works pretty cleanly since config_parser.Load interface already takes a reader anyways :)